selfdrive/car/toyota/interface.py

```diff
@@ -267,7 +267,9 @@ def get_params(candidate, fingerprint=gen_empty_fingerprint(), car_fw=[]):  # py
     # intercepting the DSU is a community feature since it requires unofficial hardware
     ret.communityFeature = ret.enableGasInterceptor or ret.enableDsu or smartDsu
 
-    if ret.enableGasInterceptor:
+    if ret.enableGasInterceptor and candidate in NO_AUTO_RESUME_CAR:
+      set_long_tune(ret.longitudinalTuning, LongTunes.TSS)
```

Comment:
why was this added?

Reply:
I didn't quite get your "move the new condition in the body up here", so I changed it this way

Reply:
just delete this new branch

Reply:
done